contrib/start_development_worker

```diff
@@ -8,7 +8,11 @@ function _print_syntax() {
   me=`basename "$0"`
   echo "Usage: $me [-d <dir> -w <num>]"
   echo -e "\t-d <dir>\tDirectory of source code. (default: /obs)"
+  echo -e "\t-l <dir>\tDirectory where the logfiles should be put. If not set STDERR and STDOUT will be printed to console"
   echo -e "\t-w <num>\tNumber of workers that should be spawned. (default: 2)"
+  echo -e "\t-h <string>\tSet hostname of server. (default: backend)"
+  echo -e "\t-r <server>\tRegister to this reposerver. Can be used multiple times."
```

Comment:
💭 just out of curiosity, what's the usecase of this? You run multiple repo servers in your development environment? How? Or is this also about pointing things to localhost?

Reply:
You can share worker between multiple backends. This is be done with the --repo switch.